gym/f110_gym/envs/f110_env.py

```diff
@@ -311,6 +322,48 @@ def reset(self, poses):
         obs, reward, done, info = self.step(action)
         return obs, reward, done, info
 
+    def add_obstacles(self, n_obstacles=4, obstacle_size=[0.5, 0.5]):
+        """
+        Adds a set number of obstacles to the envioronment. 
+        Updates the renderer and the map kept by the laser scaner for each vehicle in the simulator
+
+        Args:
+            n_obstacles (int): number of obstacles to add
+            obstacle_size (list(2)): rectangular size of obstacles
+            
+        Returns:
+            None
+        """
+        if self.renderer is None:
```

Comment:
Probably shouldn't be called here. The user needs to have control over whether rendering is on or not.

Reply:
Valid, will change